extensions/pathfactory/pathfactory-content-track.schema.json

```diff
@@ -0,0 +1,236 @@
+{
+  "meta:license": [
+    "Copyright 2023 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/pathfactory-contenttrack",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "title": "PathFactory Content Track Fieldgroup Details - V1.1",
+  "type": "object",
+  "description": "PathFactory Content Track Schema Field",
+  "definitions": {
+    "pfContentTrackFields": {
+      "properties": {
+        "pf:pathfactoryContentTrack": {
```

Comment:
Please add a complete name - pathfactory in the namespace instead of abbreviations.

Reply:
@anandphatak